### DISCUSSION TITLE
I want to install it for elementary OS 5.1.7 Hera (Ubuntu 18.04.4 LTS)

### DISCUSSION THREAD

**Discussion by ghost** [Prince781/vala-language-server#198] [Q&A] [answered]
I can't find this package for Ubuntu 18.04.4 LTS . I see this package for xUbuntu 20.04 or higher.

How can i install it on elementary os ??

**Comment by Prince781** [comment, marked as answer]
Because eOS 5.1's packages are quite old, you have to install a later version of Vala from this PPA like this: `sudo add-apt-repository ppa:vala-team/next`.

Vala 0.48 or greater is a hard dependency. I've tried to make sure that VLS can work with the other libraries it needs that also have old versions on eOS 5.1.

Incidentally, eOS 6 has Vala 0.48, so you could upgrade to that, but that's still in beta.
